script/check-size

```diff
@@ -212,10 +219,14 @@ size_nrf52840_version()
 
     for file in "${bins[@]}"; do
         "${reporter}" size "${OT_TMP_DIR}"/a/"${build_dir}"/bin/"${file}"_"${thread_version}" "${OT_TMP_DIR}"/b/"${build_dir}"/bin/"${file}"_"${thread_version}"
+        echo nm_size "${OT_TMP_DIR}"/a/"${build_dir}"/bin/"${file}"_"${thread_version}" "${OT_TMP_DIR}"/b/"${build_dir}"/bin/"${file}"_"${thread_version}"
```

Comment:
Is the `echo` correct?

----

I checked the output of the "size" GH workflow for this PR. It shows
```
nm_size /tmp/ot-size-report/a/build/lib/libopenthread-cli-ftd.a_1.2 /tmp/ot-size-report/b/build/lib/libopenthread-cli-ftd.a_1.2
5037
nm_size /tmp/ot-size-report/a/build/lib/libopenthread-cli-mtd.a_1.2 /tmp/ot-size-report/b/build/lib/libopenthread-cli-mtd.a_1.2
5038
nm_size /tmp/ot-size-report/a/build/lib/libopenthread-ftd.a_1.2 /tmp/ot-size-report/b/build/lib/libopenthread-ftd.a_1.2
5039
nm_size /tmp/ot-size-report/a/build/lib/libopenthread-mtd.a_1.2 /tmp/ot-size-report/b/build/lib/libopenthread-mtd.a_1.2
5040
nm_size /tmp/ot-size-report/a/build/lib/libopenthread-ncp-ftd.a_1.2 /tmp/ot-size-report/b/build/lib/libopenthread-ncp-ftd.a_1.2
5041
nm_size /tmp/ot-size-report/a/build/lib/libopenthread-ncp-mtd.a_1.2 /tmp/ot-size-report/b/build/lib/libopenthread-ncp-mtd.a_1.2
5042
nm_size /tmp/ot-size-report/a/build/lib/libopenthread-rcp.a_1.2 /tmp/ot-size-report/b/build/lib/libopenthread-rcp.a_1.2
5043
nm_size /tmp/ot-size-report/a/build/lib/libopenthread-radio.a_1.2 /tmp/ot-size-report/b/build/lib/libopenthread-radio.a_1.2
```

Reply:
this `echo` shows what we are comparing, because there are multiple files. There's no difference because the size doesn't change. Here's a preview:
https://github.com/openthread/openthread/runs/2137047318?check_suite_focus=true